### PR TITLE
Added synthetics support in client and server

### PR DESCRIPTION
**Pull request by mateuszmidor**
KNTK-194, KNTK-223

### CODE REVIEW

**Review by mmac-m3a — APPROVED** [review]
Good to go, but address my comments in the next PR.

File: apiv6/examples/synthetics/mesh_test/main.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	return w
+}
+
+func formatLatency(metrics *synthetics.V202101beta1MeshMetrics) string {
+	latencyMS, err := strconv.ParseInt(*metrics.GetLatency().Value, 10, 64)
+	if err != nil {
+		return "error"
+	}
+
+	return strconv.FormatInt(latencyMS/1000, 10) + "ms"
```
[the remark this answers is not in the document]
Comment (mateuszmidor) [reply apiv6/examples/synthetics/mesh_test/main.go:97]: Latency is returned in thousands of ms, so eg. 250ms is returned as 250000.

[the remark this answers is not in the document]
Comment (mateuszmidor) [reply apiv6/examples/synthetics/mesh_test/main.go:97]: Renaming the variable to just latency and adding comment

**Comment by mateuszmidor** [comment]
Follow up PR addressing the comments: https://github.com/kentik/community_sdk_golang/pull/36

**Pull request merged**